java/src/apps/gui3/ConfigBundle.java

```diff
@@ -0,0 +1,82 @@
+// ConfigBundle.java
```

Comment:
Since the methods in apps.ConfigBundle are public, not package protected, this class is not needed.
Note that apps.ConfigBundle does not follow the rules or conventions for _package_.Bundle, so its use is not limited to a single package.